publisher-provider/publisher-provider-app/src/main/java/nl/idgis/publisher/provider/database/AbstractDatabaseCursor.java

```diff
@@ -0,0 +1,104 @@
+package nl.idgis.publisher.provider.database;
+
+import akka.actor.Props;
+import akka.event.Logging;
+import akka.event.LoggingAdapter;
+import nl.idgis.publisher.provider.database.messages.AbstractDatabaseColumnInfo;
+import nl.idgis.publisher.provider.database.messages.FetchTable;
+import nl.idgis.publisher.provider.protocol.Record;
+import nl.idgis.publisher.provider.protocol.Records;
+import nl.idgis.publisher.stream.StreamCursor;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutorService;
+
+public class AbstractDatabaseCursor extends StreamCursor<ResultSet, Records> {

```

Comment:
class heeft 'Abstract' in de naam maar is niet abstract

Reply:
Opgelost